### ISSUE TITLE
Reminder doesn't work

### ISSUE THREAD

**Issue by Triniiix** [techoholic/bss#1]
I don't know why, but I can't get the reminder.bat file to work. I tried running it and instead of doing anything it displays nothing for a short time before saying "Press any key to continue..." and doing so will cause the window to close. The sequencer works perfectly fine for me. I downloaded the latest version of Python with PATH so that it'd work for command prompts, I ran as administrator, it's just the reminder.bat file that doesn't work at all whatsoever. I'm on a 64-bit Windows 10 computer. Am I doing something wrong?

**Comment by techoholic** [comment]
Hi Triniiix! 

The reminder is a very experimental program at this point; especially that version. I am currently working on turning the sequencer into a GUI interface, and this will also include a working version of the reminder. Honestly, I didn't expect anyone to actually use the reminder, but now that I know that there is at least an iota of interest, I will resume development! 

TLDR, no, you are not doing anything wrong, the reminder is just built in a non-compatible way. If you are specifically looking for a Bee Swarm Simulator timer, I recommend BeeSwarmSimulatorTimer by PlattinumBees. Thank you for leaving this comment :) I hope you find some use for the sequencer, and I'll do my best amidst my busy school life to work on the reminder.

**Comment by Triniiix** [comment]
Thank you for responding! I understood that this was in development, I just thought that I was doing something wrong so thank you for clarifying on that. I'll use that timer you recommended in the meantime and look forward to any development you make on this, good luck!
